### ISSUE TITLE
Need to initialize more settings to avoid backward text

### ISSUE THREAD

**Issue by mjstn** [UbiquityRobotics/oled_display_node#17]
We can see backwards text and other issues because there are several default settings that we do not setup as of today.

Need to go back in and do more coverage on the display settings to leave no assumptions that a reset set them up.

**Comment by anfederman** [comment]
On Noetic dev image on boot wifi IP displayed, but no battery. After LS
lidar installed, Ethernet IP, still no battery.

**Comment by mjstn** [comment]
No battery is different issue, lets not worry about that for this issue.  
I am working on a pull request to do more robust determination of led type.  It is a bit tricky and think on powerup this tends to happen but if you do a shutdown -r now it may be ok.

**Comment by mjstn** [comment]
This turned out to be the auto-detection of the display type is not reliable so we initialized for wrong type of display.  Because we have a hard spec on the chip used for the display in production the node has been changed to ALWAYS be the type of chip used in production.   Thus I close this issue.  I have tested this on one particular OLED display that was just about half the time being detected improperly and it is always working now.
